src/components/SideNav/SideNav.styles.ts

```diff
@@ -1,4 +1,7 @@
-import { makeStyles, createMuiTheme } from "@material-ui/core/styles";
+import {
+  makeStyles,
+  unstable_createMuiStrictModeTheme as createMuiTheme,
```

Comment:
hmm seems wrong to use the `unstable_createMuiStrictModeTheme` 🤔

Reply:
yeah.. seems it is still an experimental feature
We are in v4 now and this is planned to be fixed in v5